### PR TITLE
🟢 New PostMapping to add new books 

### PR DESCRIPTION
**Pull request by Sakelig**
:)

### CODE REVIEW

File: src/main/java/no/academy/bookservice/Book/BookController.java [hunk 1]
```diff
@@ -29,4 +27,11 @@ public ResponseEntity<Book> getById(@PathVariable long id) throws BookNotFoundEx
             throw new BookNotFoundException();
         }
     }
+
+
+    @PostMapping("/book")
```
Comment (larskristianhaga) [inline src/main/java/no/academy/bookservice/Book/BookController.java:32]: Can you change the path to `books`?

File: src/main/java/no/academy/bookservice/Book/BookController.java [hunk 2]
```diff
@@ -3,9 +3,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```
Comment (larskristianhaga) [inline src/main/java/no/academy/bookservice/Book/BookController.java:6]: Why this change? We only need 3 of those.

**Review by larskristianhaga — CHANGES_REQUESTED** [review]
This change is whack, change it

File: src/main/java/no/academy/bookservice/Book/BookController.java (hunk 1 above)
Comment (Sakelig) [reply src/main/java/no/academy/bookservice/Book/BookController.java:32]: oki 👍

**Comment by larskristianhaga** [comment]
You need to update the title of the PR, to better match what the change actually does.

**Review by Sakelig — COMMENTED** [review]
Fixed code with review comment 👍

File: src/main/java/no/academy/bookservice/Book/BookController.java (hunk 2 above)
Comment (Sakelig) [reply src/main/java/no/academy/bookservice/Book/BookController.java]: This was a stupid mistake 😿
